src/features/components/organisms/markets/markets-trade-steps/styles.module.scss

```diff
@@ -0,0 +1,16 @@
+@use 'features/styles/theme/theme-mixins' as *;
+.item_container {
+    background-color: var(--color-white);
+    box-shadow: rgba(14, 14, 14, 0.1) 0px 4px 8px 0px;
+    inline-size: 31.6rem;
+    padding: 2.4rem;
+    block-size: 16.4rem;
+    position: relative;
+    border-radius: 1.6rem;
+}
+
+.item_description {
+    @include breakpoints(tablet) {
+        min-block-size: 10rem;
```

Comment:
Should we use this property for desktop as well?

Reply:
okay